dashboard/app/views/levels/_level_group.haml

```diff
@@ -46,13 +46,27 @@
         = render partial: 'levels/free_response', locals: {index: index, level: level, last_attempt: last_attempt}
 
   - unless @script_level.nil?
+    - if page.page_number > 1
+      %a.btn.btn-large.btn-primary.previousPageButton
+        =t('previous_page')
+      &nbsp;
+      &nbsp;
     - if page.page_number < @total_page_count
       %a.btn.btn-large.btn-primary.nextPageButton
         =t('next_page')
     - else
       = render partial: 'levels/dialog', locals: {app: app, data: data}
-      %a.btn.btn-large.btn-primary.unsubmitButton{style: "display: none"}
-        =t('unsubmit')
+      .level_group_buttons
+        %a.btn.btn-large.btn-primary.unsubmitButton{style: "display: none"}
+          =t('unsubmit')
```

Comment:
Can this go in `_dialog.html.haml` behind `@level.properties['submittable']`?  I'd like to re-use this in `FreeResponse` if possible.